aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -299,47 +306,137 @@ module aptos_framework::account {
         account_public_key_bytes: vector<u8>,
         recipient_address: address,
     ) acquires Account {
+        offer_rotation_capability(account, rotation_capability_sig_bytes, ED25519_SCHEME, account_public_key_bytes, recipient_address);
+    }
+
+    /// Accept rotation capability from `offerer_address` if there's an existing rotation capability offer to the account owner in the offerer's account
+    public fun accept_rotation_capability_ed25519(account: &signer, offerer_address: address) : RotationCapability acquires Account {
```

Comment:
Let's separate out the rotation functionality to another PR. As this creates other issues for us to consider.

Also there's nothing about ed25519 key here.

Reply:
will move all rotation cap-related changes to another pr - but I think this function already exists on the main branch, and removing it will cause back-ward incompatibility? not sure if people are using this tho

Reply:
I guess we did, I guess we did... then I guess this is what it is...

let's contemplate if we want to put an abort here for now... but in another PR.